run_dir/static/js/project_samples.js

```diff
@@ -802,11 +813,16 @@ function load_samples_table(colOrder) {
 
                   // Remove the X from initial QC initials
                   else if(column_id == 'initials'){
-                    var sig = validation_data[column_id];
-                    if(sig.length == 3 && sig[2] == 'X'){
-                      sig = sig.substring(0,2);
+                    if(!(validation_data[column_id] == '-')){
```

Comment:
Why not use `!=` or `!==`? Would be easier to read

Reply:
True! I was just re-writing the code over and over again and forgot to clean it up a bit 😏